content/zh/seax/demo_videos_zh/videos.md

```diff
@@ -45,10 +45,4 @@ SeaX可以與Salesforce自動集成，提供"click-to-dial"等功能。同時，
 * 自動觸發客戶CSAT滿意度短信調查
 * CSAT滿意度調查列表
 
-   <iframe width="100%" height="400" src="https://www.youtube.com/embed/TsoiqFfPD8E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;"></iframe>
-
-## SeaMeet Copilot
-
-透過 SeaMeet 分析面板查看通話和會議的相關數據和分析。若想客製化面板，SeaMeet提供可自定義的 AI 效能指標，進一步了解客服人員的表現。
-
-   <iframe width="100%" height="400" src="https://www.youtube.com/embed/QdA7l8F6LBk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;"></iframe>
+   <iframe width="100%" height="350px" src="https://www.youtube.com/embed/TsoiqFfPD8E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;"></iframe>
```

Comment:
If i click on the video, it doesn't show me the whole playlist. which is here: https://www.youtube.com/watch?v=TsoiqFfPD8E&list=PL8K7_LTqly47YfmlquZMLvpLmFocvSFTb&index=6&ab_channel=SeasaltAI

can you also include the playlist link? here's an example: https://github.com/seasalt-ai/wiki/blob/75e5e593f2cf19aa3380aba6fe5afe1c90621728/content/en/seax/seax_intro/01-seax-features.md?plain=1#L44C42-L44C42